xdsl/dialects/stencil.py

```diff
@@ -345,6 +346,29 @@ def get_rank(self) -> int:
         assert isa(res_type, TempType[Attribute])
         return res_type.get_num_dims()
 
+    def get_accesses(self) -> Iterable[AccessPattern]:
+        """
+        Return the access patterns of each input.
+
+        An access is a tuple if offsets: tuple[int, ...]
+        An access pattern is a sequence of accesses: Sequence[tuple[int, ...], ...]
+        This method returns an access pattern for each stencil field the apply operates
+        on.
```

Comment:
I would love to use "offset" rather than "access" to stick closer to the dialect's terminology, or do you find it confusing?

Reply:
As in the individual offsets; not the access pattern. I like that name 👍